packages/databricks-sdk-js/src/config/oauth/Issuer.ts

```diff
@@ -0,0 +1,76 @@
+import {Config} from "../Config";
+import {Client, ClientOptions} from "./Client";
+import fetch from "node-fetch-commonjs";
+
+export class Issuer {
+    constructor(
+        public readonly config: Config,
+        public readonly authorizationEndpoint: URL,
+        public readonly tokenEndpoint: URL
+    ) {}
+
+    getClient(options: ClientOptions): Client {
+        return new Client(this, options);
+    }
+
+    static async discover(config: Config): Promise<Issuer | undefined> {
```

Comment:
this logic is usually on Config and called `getOidcEndpoints`

Reply:
moved it over